uint8 DEFAULT_TARGET=0
uint8 POSE_STAMPED=1
uint8 NAMED_TARGET_POSE=2
uint8 JOINT_STATE=3

uint8 goal_target

string robot_name
string planning_group
string named_target
geometry_msgs/PoseStamped target_pose
float64[] joint_state_target

---

bool success
